DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py

```diff
@@ -30,16 +30,6 @@
                                        DQMMessageLogger )
 
 from DQMOffline.Muon.muonMonitors_cff import *
-diMuonHistos.etaBin = cms.int32(70) #dimuonhistograms mass, bin
-diMuonHistos.etaBBin = cms.int32(70)
-diMuonHistos.etaEBin = cms.int32(70)
-diMuonHistos.etaBinLM = cms.int32(12)
-diMuonHistos.etaBBinLM = cms.int32(12)
-diMuonHistos.etaEBinLM = cms.int32(12)
-diMuonHistos.LowMassMin = cms.double(2.0)
-diMuonHistos.LowMassMax = cms.double(14.0)
-diMuonHistos.HighMassMin = cms.double(55.0)
-diMuonHistos.HighMassMax = cms.double(125.0)
```

Comment:
is there an 81X PR with cleanup of this file?
Your currently pending PR
still has these modifications
https://github.com/cms-sw/cmssw/pull/16288/files#diff-59b63a7bb9dd6c9e55764b5600006fdbR34

BTW, as I mentioned in the other PR, if these are needed for HeavyIons scenario in general and should override what we have in the pp scenario defaults, then these edits have to stay here.